### ISSUE TITLE
Account not verifyed

### ISSUE THREAD

**Issue by joshimosh** [586837r/node-red-contrib-alexa-remote2#4]
I followed the instructions:
- dragged an alexa sequence node into my flow
- created a new account and entered email and password for my alexa account
- entered service host and page for de
- entered the serial number of my Echo Dot
Always get a message "Error: Login unsuccessfull. Please check credentials."
Checked credentials a zillion times: both are correct. 
Any suggestions what might be the problem ?

**Comment by 586837r** [comment]
Thank you for the report.
I am getting the same error. The current method to login with email and password doesn't work. It does work with a cookie though.  
I am working on a way to make this work with email and password.

**Comment by joshimosh** [comment]
Thank you for the information.
Cookie did not work for me either. I used the method described here:
https://www.gehrig.info/alexa/Alexa.html
but the cookie I get has a quite different structure (does not end with csrf=xxxxxxx).
The message I get using this cookie is "Initialising", but nothing happens ...

**Comment by joshimosh** [comment]
Me again :-)
I followed you hints how to get a cookie on Firefox. Tried to copy (mark everything in Cookie window, then ctrl-c) into the cookie-filed of the account node (ctrl-v). Saved and tried again. Now I get the message:
"Error: no csrf found"

Please advise.
Thank you very much in advance.

**Comment by 586837r** [comment]
For me the cookies the first entries are `session-id`, `ubid-abcde`, `csrf` and then some.
Maybe try getting the cookie from a different request. Just like in the link you mentioned. Getting the cookie like it's described there works for me. 
I have no idea why this doesn't work for you.
If you are using Chrome you can see the individual cookie entries by clicking the lock on the left of the url. You should see `csrf` under amazon.de.   
Maybe try it again with your browser in icognito mode and with extensions/addons disabled.

Currently the node keeps trying to Intialize if you provided a faulty cookie. #5

**Comment by joshimosh** [comment]
I found (and was able to fix) the problem. In Firefox, when I select the cookie and copy it, I get a structure like:
...
session-id-time<tab>2175143249
csrf<tab>0123456789
...
...

When I manually edited this and changed it to
...
session-id-time=2175143249;
csrf=0123456789;
...
...
(for each of the - in my case - 8 lines)
it worked for me.
I am using Firefox Quantum 63.0.3 (64-Bit) on a Windows 10 system.
